bin/riemann-proc

```diff
@@ -7,9 +7,9 @@ require File.expand_path('../../lib/riemann/tools', __FILE__)
 class Riemann::Tools::Proc
   include Riemann::Tools
 
-  opt :proc_regex, "regular expression that matches the process to be monitored", type: :string
-  opt :proc_min_critical, "running process count minimum", :default => 1
-  opt :proc_max_critical, "running process count maximum", :default => 1
+  opt :proc_regex, "regular expression that matches the process to be monitored", type: :string, :default => ".*"
+  opt :proc_min_critical, "running process count minimum", :default => 0
```

Comment:
Probably worth adding something to the commit msg explaining this change in case anyone relies on the logic?